### PR TITLE
Create Google Summer of Code 2023 webpage

### PR DESCRIPTION
**Pull request by Echolon**
- [x] Create English page
- [ ] Add localizations
- [x] Create Flyers
- [x] Update Images

### CODE REVIEW

File: content/community/gsoc-2023.de.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+Title: XSF und Google Summer of Code 2023
```
Comment (wurstsalat3000) [inline content/community/gsoc-2023.de.md:2]:
```suggestion
title: XSF und Google Summer of Code 2023
```

File: content/community/gsoc-2023.de.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+![XSF and GSoC 2022 Logo](/images/logos/GSoC_2022_Logo.png)
+
+_Zur Kenntnis: Die Hauptkommunikation zu dem Thema findet auf englisch statt und nur die englischen Informationen gelten als offiziell._
+
+Die XMPP Standards Foundation (XSF) plant, als gastgebende Organisation am [Google Summer of Code 2023](https://summerofcode.withgoogle.com/programs/2023) teilzunehmen.
+Wenn Sie daran interessiert sind, als Student (oder einfach nur als Open-Source-Neuling), Mentor oder XMPP-Projekt teilzunehmen, ist dies die Seite, mit der du beginnen kannst!
```
Comment (wurstsalat3000) [inline content/community/gsoc-2023.de.md:10]:
```suggestion
Wenn du daran interessiert bist, als Student (oder einfach nur als Open-Source-Neuling), Mentor oder XMPP-Projekt teilzunehmen, ist dies die Seite, mit der du beginnen kannst!
```

File: content/community/gsoc-2023.de.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+ 
+Wenn du dir vorstellen kannst, während des Sommers in Teilzeit (oder sogar Vollzeit) an einem Open-Source-Softwareprojekt zu arbeiten, dann sollten Sie sich überlegen, an einem XMPP-Kommunikationsprojekt mitzuwirken und am Google Summer of Code 2022 teilzunehmen!
```
Comment (wurstsalat3000) [inline content/community/gsoc-2023.de.md:12]:
```suggestion
Wenn du dir vorstellen kannst, während des Sommers in Teilzeit (oder sogar Vollzeit) an einem Open-Source-Softwareprojekt zu arbeiten, dann solltest du dir überlegen, an einem XMPP-Kommunikationsprojekt mitzuwirken und am Google Summer of Code 2022 teilzunehmen!
```

File: content/community/gsoc-2023.de.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+### Ressourcen und Einstiegsmöglichkeiten
+
+Wenn ihr ein interessierter Beitragender oder ein interesiertes Projekt seid, solltet ihr ersten beiden Links unbedingt lesen! Eine frühzeitige Einbindung und Kommunikation mit uns ist für eine fruchtbare Zusammenarbeit unerlässlich.
+
+- Lies die Ressourcen auf [Googles Website] (https://summerofcode.withgoogle.com/help) und prüfe, ob dies etwas für dich und Ihre Pläne in diesem Jahr ist.
+- Wirf einen ersten Blick auf unsere XMPP-Projektdetailseite im [XSF-Wiki](https://wiki.xmpp.org/web/Google_Summer_of_Code_2023).
+- Du kannst uns einfach [per Web-Chat](https://xmpp.org/chat#converse/room?jid=gsoc@muc.xmpp.org) erreichen oder die ["Getting started with XMPP"](https://xmpp.org/getting-started/) Seite durchsehen und uns über unseren [öffentlichen Chat](xmpp:gsoc@muc.xmpp.org?join) erreichen.
+
+Schauen dir auch unsere Medienkanäle an!
```
Comment (wurstsalat3000) [inline content/community/gsoc-2023.de.md:42]:
```suggestion
Schaue dir auch unsere Medienkanäle an!
```

File: content/community/gsoc-2023.en.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+Title: XSF and Google Summer of Code 2023
```
Comment (wurstsalat3000) [inline content/community/gsoc-2023.en.md:2]:
```suggestion
title: XSF and Google Summer of Code 2023
```

File: content/community/gsoc-2023.en.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+XMPP is the Extensible Messaging and Presence Protocol, a set of open technologies for instant messaging, presence, multi-party chat, voice and video calls, collaboration, lightweight middle-ware, content syndication, and generalized routing of XML data. XMPP was originally developed in the Jabber open-source community to provide an open, decentralized alternative to the closed instant messaging services at that time.
+
+### Google Summer of Code 2023
+
+The Google Summer of Code is an international annual program in which Google awards stipends to contributors who successfully complete a free and open-source software coding project during the summer. Project ideas are listed by host organizations involved in open-source software development, though students can also propose their own project ideas. 
+
+### Participating projects
+
+**The XSF has not yet applied nor being accepted!**
```
Comment (wurstsalat3000) [inline content/community/gsoc-2023.en.md:21]:
```suggestion
**The XSF has not applied yet and thus hasn't been accepted yet!**
```

**Comment by Echolon** [comment]
@wurstsalat3000 can you help me doing an announcement and link the correct page under _Community_?

**Pull request merged**